src/java.base/unix/classes/java/lang/ProcessImpl.java

```diff
@@ -100,16 +100,24 @@ private static LaunchMechanism launchMechanism() {
             // Should be value of a LaunchMechanism enum
             LaunchMechanism lm = LaunchMechanism.valueOf(s.toUpperCase(Locale.ROOT));
             switch (OperatingSystem.current()) {
-                case LINUX:
-                    return lm;      // All options are valid for Linux
+                case LINUX: {
+                    // All options are valid for Linux, but VFORK is deprecated and results
+                    // in a warning
+                    if (lm == LaunchMechanism.VFORK) {
+                        System.err.println("VFORK MODE DEPRECATED");
+                        System.err.println("The VFORK launch mechanism has been deprecated for being dangerous.\n" +
+                                           "It will be removed in a future java version. Either remove the\n" +
+                                           "jdk.lang.Process.launchMechanism property (preferred) or use FORK mode\n" +
+                                           "instead (-Djdk.lang.Process.launchMechanism=FORK).");
```

Comment:
This is okay although I could imagine someone wanting to replace this with a text block.

Reply:
okay, changed to string block